tests/scripts/thread-cert/Cert_5_1_06_RemoveRouterId.py

```diff
@@ -100,27 +100,32 @@ def test(self):
         msg = router1_messages.next_coap_message("0.02")
         msg.assertCoapMessageRequestUriPath("/a/as")
 
-        msg = leader_messages.next_coap_message("2.04")
+        leader_messages.next_coap_message("2.04")
 
         # 3 - Router1
-        router1_messages.next_mle_message(mle.CommandType.PARENT_REQUEST)
-        leader_messages.next_mle_message(mle.CommandType.PARENT_RESPONSE)
+        msg = router1_messages.next_mle_message(mle.CommandType.PARENT_REQUEST)
+        msg.assertSentWithHopLimit(255)
```

Comment:
use existing `check_parent_request()` function?
https://github.com/openthread/openthread/blob/master/tests/scripts/thread-cert/command.py#L237

one suggestion is to add parameter for `check_parent_request()` to indicate whether this is first parent request attempt (scan mask should be 0x80) or second attempt (scanmask 0xc0).

Reply:
done. 
The test plan document requires only checking if scanMakk TLV presents in the message,  so doesn't add this parameter.